import/keycloak-themes/catenax-central/login/resources/js/Main.js

```diff
@@ -48,7 +48,7 @@ const N = (tag, c, att) => {
     return append(n, c)
 }
 const SEARCH_VALIDATION_REGEX =
-  /^[a-zA-Z0-9][a-zA-Z0-9 !#'$@&%()*+,\-_./:;=<>?[\]\\^]{0,255}$/
+  /^[a-zA-ZÀ-ÿŚął\d][a-zA-ZÀ-ÿŚął\d !#'$@&%()*+,\-_./:;=<>?[\]\\^]{0,255}$/;
```

Comment:
; is not required here
```suggestion
  /^[a-zA-ZÀ-ÿŚął\d][a-zA-ZÀ-ÿŚął\d !#'$@&%()*+,\-_./:;=<>?[\]\\^]{0,255}$/
```